### PR TITLE
Refactor styleguide

### PR DESCRIPTION
**Pull request by tonifisler**

### CODE REVIEW

File: drupal/sites/all/themes/antistatique/assets/components/atoms/button.html.swig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Button
+name: button
+collection: atoms
+variants:
+  primary: Primary
```
Comment (jminguely) [inline drupal/sites/all/themes/antistatique/assets/components/atoms/button.html.swig]: This variants isn't used & could be defined as useless 🤡

File: drupal/sites/all/themes/antistatique/assets/docs/javascript.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+title: JavaScript
+collection: docs
+styleguide: true
+heading_color: '#ff0'
+---
+
+This is a markdown document.
+
+```javascript
+var foo = 'bar'
```
Comment (jminguely) [inline drupal/sites/all/themes/antistatique/assets/docs/javascript.md]: Useful ?

File: drupal/sites/all/themes/antistatique/assets/sass/atoms/_button.scss [hunk 3]
```diff
@@ -190,13 +200,13 @@
...
 
 .btn-xxs {
-  font-size: .8em;
+  font-size: 0.8em;
   line-height: 1;
 
   @media only screen and (min-width: $screen-sm-min) {
-    font-size: .6em;
-    line-height: 1;
     padding: 5px 6px 4px;
```
Comment (jminguely) [inline drupal/sites/all/themes/antistatique/assets/sass/atoms/_button.scss]: It's more a question than a review : 
shouldn't it be better to use `em` units for the padding ? We already use these relative value for font-size and line-height.

File: drupal/sites/all/themes/antistatique/assets/sass/atoms/_heading.scss [hunk 4]
```diff
@@ -1,58 +1,148 @@
...
+
+  h1,
+  .h1,
+  h2,
+  .h2,
+  h3,
+  .h3 {
     small,
     .small {
       font-size: 65%;
```
Comment (jminguely) [inline drupal/sites/all/themes/antistatique/assets/sass/atoms/_heading.scss]: Why use percentage here ? Could we achieve that with `em` units also ?

File: drupal/sites/all/themes/antistatique/assets/sass/others/_coffee.scss [hunk 5]
```diff
@@ -7,74 +7,78 @@
...
-    img {width: 230px;}
+
+    @media only screen and (max-width: $screen-xs-max) { margin-top: 10px; }
+    img { width: 230px; }
   }
 
   .christmas-tree {
     text-align: center;
     margin: auto 0;
     padding: 0 0;
```
Comment (jminguely) [inline drupal/sites/all/themes/antistatique/assets/sass/others/_coffee.scss]: Stylint advises us to use `padding: 0;`

File: drupal/sites/all/themes/antistatique/assets/sass/atoms/_heading.scss (hunk 4 above)
Comment (tonifisler) [inline drupal/sites/all/themes/antistatique/assets/sass/atoms/_heading.scss]: Bootstrap uses percentage for the `.small` element. Not sure why...

File: drupal/sites/all/themes/antistatique/assets/docs/javascript.md (hunk 2 above)
Comment (tonifisler) [inline drupal/sites/all/themes/antistatique/assets/docs/javascript.md]: Haha yes!! 😁

File: drupal/sites/all/themes/antistatique/assets/components/atoms/alert.html.swig [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+collection: atoms
+variants:
+  default: Danger
+  success: Success
+  info: Info
+  warning: Warning
+notes: |
+  Standard alert component.
+---
+<div class="alert alert-{% if variant %}{{variant}}{% else %}danger{% endif %}">
```
Comment (tonifisler) [inline drupal/sites/all/themes/antistatique/assets/components/atoms/alert.html.swig]: 😱 never thought of that haha

File: drupal/sites/all/themes/antistatique/assets/sass/atoms/_button.scss (hunk 3 above)
Comment (tonifisler) [inline drupal/sites/all/themes/antistatique/assets/sass/atoms/_button.scss]: This is some old code, I won't change it back. Hoping on a redesign to happen shortly 😄

**Comment by tonifisler** [comment]
Once this is merged, we need to paste this in https://antistatique.net/fr/admin/structure/block/manage/block/12/configure and in https://antistatique.net/fr/admin/structure/block/manage/block/13/configure

```html
<div class="spacer spacer-sm"></div>
<div class="container">
  <div class="row">
    <div class="col-sm-12 text-center">
      <div class="list-clients">
        <div class="client">
          <img class="img-responsive" alt="Canton de Genève" src="/sites/all/themes/antistatique/build/svg/canton-de-geneve.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Canton de Vaud" src="/sites/all/themes/antistatique/build/svg/canton-de-vaud.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Confédération Suisse" src="/sites/all/themes/antistatique/build/svg/confederation-suisse.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Ville de Genève" src="/sites/all/themes/antistatique/build/svg/Ville-de-Geneve.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Cardis Sothebys" src="/sites/all/themes/antistatique/build/svg/Cardis-Sothebys.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="RTS" src="/sites/all/themes/antistatique/build/svg/RTS.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Le Temps" src="/sites/all/themes/antistatique/build/svg/Le-Temps.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Vaudoise" src="/sites/all/themes/antistatique/build/svg/Vaudoise.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="AXS" src="/sites/all/themes/antistatique/build/svg/AXA.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="EHL" src="/sites/all/themes/antistatique/build/svg/EHL.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Loterie Romande" src="/sites/all/themes/antistatique/build/svg/Loterie-Romande.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Médecins sans Frontières" src="/sites/all/themes/antistatique/build/svg/Medecins-sans-Frontieres.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="Orbium" src="/sites/all/themes/antistatique/build/svg/Orbium.svg">
        </div>
        <div class="client">
          <img class="img-responsive" alt="United World Wrestling" src="/sites/all/themes/antistatique/build/svg/UWW.svg">
        </div>
      </div>
    </div>
  </div>
</div>
```

**Pull request merged**